### PR TITLE
Edit GitHub Issues Page (fixes #2053)

### LINKED ISSUE #2053: Better organization of GitHub Issues article (Step 6)
### Problem
GitHub Issues article has too many headers which can be overwhelming for the reader.

### Steps to reproduce the problem
N/A

### Screenshots
step_6

### Proposed solution
Reorganize content, substitute some headers to subheaders and make emphasis on the main points of the article.

The structure can be next:
# GitHub Issues
## Create a New Branch
* Preview Changes Locally
## Create a Commit and Push the Changes
* Commit Message Style Guide
* Rawgit
   * Rawgit Generator
## Useful Links

### PR DESCRIPTION
**Pull request by svlesiv**
Fixes #2053

### Description
Structure of the page has been altered. Also, I made some pieces of the content more concise.

### RawGit preview link
https://rawgit.com/svlesiv/svlesiv.github.io/issues_change_structure_branch/#!./pages/vi/vi-github-issues.md

### CODE REVIEW

File: pages/vi/vi-github-issues.md [hunk 1]
```diff
@@ -34,7 +34,12 @@ _**SPECIAL NOTE:**_ Aspiring Virtual Interns have a tendency to speed through th
 
 ## Create a New Branch
 
-Every time you start to work on an issue, you need to create a branch to keep issues you are working on separated from each other. Before you create your branch, sync your repo following the process in Summary of Steps - Sync Your Fork section of the [previous step](vi-github-and-repositories.md#Summary_of_Steps).
+Every time you start to work on an issue, you need to create a branch to keep issues you are working on separated from each other. Before you create your branch, sync your repo using next commands: 
+`git fetch upstream` ,
```
Comment (lmmrssa) [inline pages/vi/vi-github-issues.md]: could you remove `,` in code

File: pages/vi/vi-github-issues.md [hunk 2]
```diff
@@ -112,24 +126,26 @@ Additionally, creating a checklist will help others understand what you have don
...
 
+* **Delete the branch from your local repository.**
+To delete the defunct branch in your local repository, use the command `git branch -d yourLocalBranchName` (make sure you are not on the branch that you want to delete). Go [here](https://tecadmin.net/delete-git-remote-and-local-branch/) to find out more about how to delete a branch locally.
+
 NOTE: Remember to always sync your fork before starting to work on a new issue. To sync your fork you can follow the process in the [previous step](vi-github-and-repositories.md).
 
 This is an exercise to help you familiarize with GitHub issues, committing, and creating pull requests. This is a common process in large open source projects as there is always room for improvement. So, we strongly encourage you to follow this process and continue to post issues and resolve them.
 
-## Summary of Steps to Work on an Issue and Create a Pull Request
+<!-- ## Summary of Steps to Work on an Issue and Create a Pull Request
```
Comment (lmmrssa) [inline pages/vi/vi-github-issues.md]: Remove the code instead of commenting

Comment (svlesiv) [reply pages/vi/vi-github-issues.md]: Sure, thank you. I forgot to do it.

File: pages/vi/vi-github-issues.md [hunk 3]
```diff
@@ -112,40 +126,23 @@ Additionally, creating a checklist will help others understand what you have don
 
 Some of these items are of course optional, but try to include as much useful information for others as possible in your checklists because nobody really wants to work on something that has already been completed by someone else.
 
-## Wait for Review
+## Delete the Branch after Staff review
```
Comment (nadaa) [inline pages/vi/vi-github-issues.md]: I prefer to make it as "Delete the Branch after your changes merged".

Comment (svlesiv) [reply pages/vi/vi-github-issues.md]: @nadaa, thank you for your suggestion. 'Delete the Branch after changes merged' better corresponds with actual workflow, and 'Delete the Branch after Staff review' can be misleading. 
However, maybe as for the title, I will leave only 'Delete the Branch,' and explain when to delete in the paragraph itself.  What do you think?

Comment (nadaa) [reply pages/vi/vi-github-issues.md]: @svlesiv, it wil be better!

**Pull request merged**